Parsnip/index.js

```diff
@@ -56,11 +56,11 @@ bot.on("message", async message => {
     let earnings = new Discord.RichEmbed()
       .setDescription("__**LINKS**__")
       .setColor("#50f442")
-      .addField("How much can I earn with salad?","https://salad.zendesk.com/hc/en-us/articles/360018704652-How-much-can-I-earn-with-Salad-")
-      .addField("How can I earn more with Salad?", "https://salad.zendesk.com/hc/en-us/articles/360019038251-How-can-I-earn-more-with-Salad-")
-      .addField("How does Salad mine cryptocurrency?", "https://salad.zendesk.com/hc/en-us/articles/360019038151-Does-Salad-mine-Cryptocurrency-");
+      .addField("How much can I earn with salad?","https://salad.zendesk.com/hc/en-us/articles/360028553551-How-much-can-I-earn-with-Salad-")

```

Comment:
```suggestion
      .addField("How much can I earn with Salad?","https://salad.zendesk.com/hc/en-us/articles/360028553551-How-much-can-I-earn-with-Salad-")
```
Just adjusting capitalization slightly.